internal/metamorph/store/postgresql/postgres_test.go

```diff
@@ -177,25 +177,23 @@ func TestPostgresDB(t *testing.T) {
 	minedData := &store.StoreData{
 		RawTx:        make([]byte, 0),
 		StoredAt:     now,
-		AnnouncedAt:  time.Date(2023, 10, 1, 12, 5, 0, 0, time.UTC),
-		MinedAt:      time.Date(2023, 10, 1, 12, 10, 0, 0, time.UTC),
 		Hash:         minedHash,
 		Status:       metamorph_api.Status_MINED,
 		BlockHeight:  100,
 		BlockHash:    testdata.Block1Hash,
 		Callbacks:    []store.StoreCallback{{CallbackURL: "http://callback.example.com", CallbackToken: "12345"}},
 		RejectReason: "not rejected",
 		LockedBy:     "metamorph-1",
+		LastModified: nil,
 	}
 
 	unminedHash := testdata.TX1Hash
 	unminedData := &store.StoreData{
-		RawTx:       make([]byte, 0),
-		StoredAt:    now,
-		AnnouncedAt: time.Date(2023, 10, 1, 12, 5, 0, 0, time.UTC),
-		Hash:        unminedHash,
-		Status:      metamorph_api.Status_SENT_TO_NETWORK,
-		LockedBy:    "metamorph-1",
```

Comment:
I see you removed `announcedAt` from every test, but you didn't add the new statusHistory to any unit test. Please add it to one, or possibly to couple of tests to see if everything is working correctly with parsing, saving, etc. 🙏